internal/kafka/internal/workers/cluster_mgrs/dynamic_scale_down_mgr.go

```diff
@@ -90,7 +90,7 @@ func (m *DynamicScaleDownManager) processDynamicScaleDownReconcileEvent() error
 		clusterID := suCount.ClusterId
 		existing := processedClusters[clusterID]
 		if existing.processed { // skip already processed
-			glog.V(10).Infof("cluster with cluster_id %s is already processed", suCount.ClusterId)
+			glog.V(10).Infof("cluster with cluster id '%q and status %q is already processed. Skipping it from scale down evaluation", suCount.ClusterId, suCount.Status)
```

Comment:
Extra `'` with `%q`. I see it in multiple places in this file

```suggestion
			glog.V(10).Infof("cluster with cluster id %q and status %q is already processed. Skipping it from scale down evaluation", suCount.ClusterId, suCount.Status)
```